### PR TITLE
Egun sprite change.

### PR DESCRIPTION
**Pull request by Kierany9**
Crystal told me to go ahead and make the damn pr already so here we are. The current sprite's inconsistency is triggering me intensely:

- Egun now glows yellow when in stun mode instead of blue to be consistent with every other weapon.
- Also fixes a spriting error where an egun with one charge would appear as if it had two in certain inhand sprites.

piss off crystal

why were these blue in the first place stuns arent blue, disablers are blue, stuns are yellow.

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
I wish we could get rid of the ugly ass outer bits
It looks nice, but I wish it was completely changed.

**Comment by Crystalwarrior** [comment]
Changelog looks ok :+1: 

Just needs a head approval and we're set

**Comment by Lawligagger** [comment]
doesn't look as bad as I thought it would +1

**Comment by trifornt-tinfornt** [comment]
I wish we still had the before rebase sprites. Those looked way better.

**Comment by dreamcrab** [comment]
i like it, despite what nebyab says. doesnt matter either way but i +1 yes A++

**Comment by carlarc** [comment]
i remember tg fixed this by making the stun a disabler
nothing interesting to add, merge this crap

**Comment by Kierany9** [comment]
Egun really should be a disabler instead of a taser but that'd upset all the powergamers. It's really useful for HoP though, weaken all the greyshits breaking your windows from inside your office.

**Comment by Nebyab** [comment]
nah, to both the sprite change and the idea that the egun of all things should now be a disabler

**Comment by Nebyab** [comment]
I'm aware its not within the purview of this PR, but its a dumb idea if you ask me

**Comment by McBawbaggings** [comment]
I'll support this if you can this pushed onto /tg/. I don't think we should be having different coloured guns from other servers for sake of server consistency.

**Comment by Kierany9** [comment]
From what I know /tg/ eguns shoot disablers which would make pushing this change there useless.

**Comment by Crystalwarrior** [comment]
1 veto from nebyab for whatever dumb reason, merge or no?

**Comment by Carbonhell** [comment]
>server consistency
wtf?who gives a fuck,it's a colour,it's not like it's gonna make anyone confused about anything
i fully agree with this pr

File: html/changelogs/Kierany9 - Egun sprites [hunk 1]
```diff
@@ -0,0 +1,36 @@
+################################
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - Egun sprites]: broken changelog,you forgot the .yml extension

**Comment by calibraptor** [comment]
+1

**Comment by Spacedong** [comment]
I don't really like it, its been this way forever and you don't need to be super smart to know that it shoots electrodes. I'd rather just have the old e-gun sprites from pre-rebase along with e-pistols for the heads.

**Comment by McBawbaggings** [comment]
If /tg/ use disablers for eguns then it makes sense for our yellow to be tazers.

**Comment by Lawligagger** [comment]
denying due to the votes against it.

**Comment by Lawligagger** [comment]
merging due to last minute mind change and his scottish grammar getting the better of him

**Comment by Crystalwarrior** [comment]
do it

**Pull request merged**